Resources/Prototypes/DeltaV/Entities/Objects/Devices/Electronics/door_access.yml

```diff
@@ -93,3 +93,27 @@
   components:
   - type: AccessReader
     access: [["Mail"]]
+    
+    - type: entity
```

Comment:
Check the indentations on these

Reply:
Resolved

Reply:
` - type: entity` still looks like it's indented too far on my end

Reply:
When you click on your review, it shows you the outdated file that was present at the time of the review. The indentation was fixed up